rotary/rules/circularize.smk

```diff
@@ -245,7 +245,7 @@ rule prepare_polypolish_circularize_input:
     input:
         "{sample}/circularize/circlator/{sample}_rotated.fasta"
     output:
-        "{sample}/circularize/polypolish/input/{sample}_input.fasta"
+        temp("{sample}/circularize/polypolish/input/{sample}_input.fasta")
```

Comment:
Why is this Polypolish input rule temp() and the other one (`prepare_polypolish_polish_input`) not temp()? I remember tinkering with these while trying to ensure that the pipeline finished without bugs related to the DAG, but I am not sure I compared them carefully. Because these are symlinks, the safe thing to do would be to keep both as non-temp, unless I have a chance to test in more detail.

Reply:
Remove the temp() on this output file now.